tests/test_cli.py

```diff
@@ -13,7 +13,10 @@
     "no_callable()",
     "no_equals = ",
     "BeamSequence[0]extra",  # must match to end of string
+    "(300a,00b0)[0]extra",  # as above
     "BeamSequence[x]",  # index must be an int
+    "(0010,0010b)",  # bad tag format
+    "BeamSequence[0].(10,10)",  # bad tag format not initial
```

Comment:
not initial?

Reply:
Yeah, somewhere in my thoughts that sounded very awkward.  Changing to 'nested' instead...